File: plugins/approvej-intellij-plugin/src/main/java/org/approvej/intellij/ApproveJIcons.java
package org.approvej.intellij;

import com.intellij.openapi.util.IconLoader;
import javax.swing.Icon;

final class ApproveJIcons {

  static final Icon APPROVED = IconLoader.getIcon("/icons/approved.svg", ApproveJIcons.class);

  static final Icon APPROVAL_PENDING =
      IconLoader.getIcon("/icons/approvalPending.svg", ApproveJIcons.class);
}

File: plugins/approvej-intellij-plugin/src/main/kotlin/org/approvej/intellij/ApproveJIcons.kt
package org.approvej.intellij

import com.intellij.openapi.util.IconLoader

internal object ApproveJIcons {

  val APPROVEJ = IconLoader.getIcon("/icons/approvej.svg", ApproveJIcons::class.java)
  val APPROVED = IconLoader.getIcon("/icons/approved.svg", ApproveJIcons::class.java)
  val APPROVAL_PENDING = IconLoader.getIcon("/icons/approvalPending.svg", ApproveJIcons::class.java)
}
